src/script/interpreter.cpp

```diff
@@ -2398,6 +2399,49 @@ bool GenericTransactionSignatureChecker<T>::CheckSequence(const CScriptNum& nSeq
     return true;
 }
 
+template <class T>
+unsigned int GenericTransactionSignatureChecker<T>::GetLockTime() const
+{
+    return txTo->nLockTime;
+}
+
+template <class T>
+unsigned int GenericTransactionSignatureChecker<T>::GetTxVersion() const
```

Comment:
Return an (u)int32_t.

Reply:
This now returns int32_t, when is later cast to uint32_t before writing on the stack.